src/i18n/messages/es_419.json

```diff
@@ -24,10 +24,11 @@
   "footer.connectLinks.blog": "Blog",
   "footer.connectLinks.contact": "Contáctenos",
   "footer.connectLinks.help": "Centro de Ayuda",
+  "footer.connectLinks.security": "Seguridad",
```

Comment:
Slightly odd workflow, but this file is generated by Transifex and checked in.  So changes here will be blown away by the next transifex job that runs.  The "correct" workflow is to just change the English source string like you did in Footer.messages.js, check that in, and wait for translators to update the spanish translation in transifex.